Plots/MapProjections/NCL_radar_1.py

```diff
@@ -0,0 +1,127 @@
+"""
+NCL_radar_1.py
+===============
+This script illustrates the following concepts:
+   - Fitting radial data to a cartesian grid
+   - Creating a vertical colorbar
```

Comment:
```suggestion
   - Creating a horizontal colorbar
```
Colorbar is horizontal not vertical

Reply:
Whoops, good catch, thanks!